src/components/Header/Header.tsx

```diff
@@ -32,12 +33,19 @@ interface Props {
   paragraph: string;
 }
 
-export default function Header({ title, ouiaId, paragraph }: Props) {
+export default function Header({ ouiaId, paragraph }: Props) {
```

Comment:
the 'title' is passed in from the repositories page and the templates page, so we probably want to continue that and just reference it via `{title}` below?